patches/minecraft/net/minecraft/inventory/SlotFurnaceOutput.java.patch

```diff
@@ -1,11 +1,18 @@
 --- ../src-base/minecraft/net/minecraft/inventory/SlotFurnaceOutput.java
 +++ ../src-work/minecraft/net/minecraft/inventory/SlotFurnaceOutput.java
-@@ -83,6 +83,8 @@
+@@ -37,6 +37,7 @@
  
+     public void func_82870_a(EntityPlayer p_82870_1_, ItemStack p_82870_2_)
+     {
++        if(field_75228_b>0)
+         this.func_75208_c(p_82870_2_);
```

Comment:
"Fix ContainerFurnace invalid call after -onSlotChange(ItemStack,ItemStack), without removing valid call after -decrStackSize(int)."
Is that good enough, or am i being too cryptic ?
Also, should i inline this instead ?